### PR TITLE
Adding new features

### PR DESCRIPTION
**Pull request by daruNBC**
# Adding new features
Added new features to the file.js and README.md just to test some little quick
[figma.txt](https://github.com/user-attachments/files/15935674/figma.txt)
n-tier

### CODE REVIEW

**Comment by daruNBC** [comment]
## OOPS! I made a mistake in that title

Added new features to the file.js and README.md just to test some little quick

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 # Demo
+
 ### small title
+
 Test text 2
+Test text 3
```
Comment (DarylMaster) [inline README.md]: why did you add this code here?

File: file.js [hunk 2]
```diff
@@ -1 +1,6 @@
-console.log("hello world")
```
Comment (DarylMaster) [inline file.js:1]: I noticed that you removed mine

Comment (DarylMaster) [reply file.js:1]: can you answer this little quick

**Review by DarylMaster — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (daruNBC) [reply README.md]: just to test sir

File: file.js (hunk 2 above)
Comment (daruNBC) [reply file.js:1]: because it didn't work sir

Comment (DarylMaster) [reply file.js:1]: sure about that?

Comment (DarylMaster) [reply file.js:1]: > because it didn't work sir

Yo man just ...

**Review by DarylMaster — APPROVED** [review]
